move/axelar_gateway/sources/gateway.move

```diff
@@ -498,7 +499,7 @@ fun test_take_approved_message_message_not_approved() {
         axelar_gateway::bytes32::new(@0x2),
     );
 
-    fields_mut!(&mut gateway)
+    gateway.inner.load_value_mut<GatewayV0>()
```

Comment:
use the `gateway.value_mut!` macro here as well?

Reply:
can't anymore cause it checks version control.

Reply:
we discussed updating the dummy method to set this up correctly